packages/api/tests/endpoints/test-granules.js

```diff
@@ -444,16 +468,16 @@ test.serial('PUT fails if action is not supported', async (t) => {
   t.true(message.includes('Action is not supported'));
 });
 
-test.serial('PUT fails if action is not provided', async (t) => {
+test.serial('PUT updates granule if action is not provided', async (t) => {
   const response = await request(app)
     .put(`/granules/${t.context.fakeGranules[0].granuleId}`)
     .set('Accept', 'application/json')
     .set('Authorization', `Bearer ${jwtAuthToken}`)
```

Comment:
Do you have to send a granule object here?

Reply:
If I don't then the test for granuleName === granuleId fails. Do you think there should be another test for that?  PUT granule/granuleName with no object I think it will fail with `input :granuleName (gran0313) must match body's granuleId (undefined)`

Reply:
we could keep it as a error case.

Reply:
db2daaa7a2